File: R/tests/testthat/test_hgf_binary_binary.R
context("Test hgf_binary_binary")
library(hBayesDM)

test_that("Test hgf_binary_binary", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(hgf_binary_binary(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_hgf_binary_binary.py
import pytest

from hbayesdm.models import hgf_binary_binary


def test_hgf_binary_binary():
    _ = hgf_binary_binary(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
